pkg/network/node/pod.go

```diff
@@ -153,7 +154,8 @@ func getIPAMConfig(clusterNetworks []common.ClusterNetwork, localSubnet string)
 		Name:       "openshift-sdn",
 		Type:       "openshift-sdn",
 		IPAM: &hostLocalIPAM{
-			Type: "host-local",
+			Type:    "host-local",
+			DataDir: hostLocalDataDir,
```

Comment:
Where is hostLocalIPAM.DataDir used?

Reply:
@pravisankar it's serialized into JSON and passed to the host-local plugin, which then puts its IPAM data there instead of the default.  In reality hostLocalDataDir is already the same as the host-local plugin default so no change.  But the path will be consistent now.